pkg/multus/multus_test.go

```diff
@@ -68,30 +68,37 @@ type fakeExec struct {
 	delIndex        int
 	chkIndex        int
 	expectedDelSkip int
-	plugins         []*fakePlugin
+	//plugins         []*fakePlugin
```

Comment:
Curious about what are your reasons to keep this around in the code .

Reply:
changed.